serviceconnector/src/createServiceConnector/ClientTypeStep.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { KnownClientType } from "@azure/arm-servicelinker";
+import { AzureWizardPromptStep, IAzureQuickPickItem } from "@microsoft/vscode-azext-utils";
+import * as vscode from 'vscode';
+import { ICreateLinkerContext } from "./ICreateLinkerContext";
+
+export class ClientTypeStep extends AzureWizardPromptStep<ICreateLinkerContext>{
+    public async prompt(context: ICreateLinkerContext): Promise<void> {
+        const placeHolder = vscode.l10n.t('Select Language/Framework');
+        const picks: IAzureQuickPickItem<KnownClientType>[] = [
```

Comment:
Is this ordering the same as the portal? It feels a little random/arbitrary.

I'd think we should either do it alphabetically or by what we expect to be the most common choices. That being said, I think it'd be cool if client extensions have a way to recommend/organize these picks.

For example, for a Function App, it would be very easy for us to see what runtime their app is running and have it be at the top.

Reply:
This is the order the portal does except they exclude PHP and Ruby. I do think that the portal has some smart detection where they will pre fill in a client type for you based on the runtime of your app which would be helpful to add.